integration/test/utils/index.ts

```diff
@@ -7,7 +7,13 @@ export * from './matcher';
 export * from './ws-promise';
 export const hello = { hello: 'world' };
 export const serviceOptionsFactory = (app: string, json = false): OgmaServiceOptions => {
-  return { application: app, stream, json };
+  return {
+    application: app,
+    stream,
+    json,
+    logPid: false,
+    logHostname: false,
```

Comment:
Any reason these are set to false in the tests?

Reply:
To showcase that it works as intended, do you want to remove it (i.e. keep defaults)?

Reply:
I think I'd rather have a dedicated test that shows this working as we expect it to, rather than modifying the existing integration suite.